content/docs/configuration/acme/dns01/route53.md

```diff
@@ -47,30 +47,383 @@ permissions:
+> to learn how cert-manager, which uses the AWS SDK for Go V2, supports all these ambient credential sources.
+>
+> ⚠️ By default, cert-manager will only use ambient credentials for
+> `ClusterIssuer` resources, not `Issuer` resources.
+>
+> This is to prevent unprivileged users, who have permission to create Issuer
+> resources, from issuing certificates using credentials that cert-manager
+> incidentally has access to.
+> ClusterIssuer resources are cluster scoped (not namespaced) and only platform
+> administrators should be granted permission to create them.
+>
+> ⚠️ It is possible (but not recommended) to enable ambient authentication mechanisms
+> for `Issuer` resources, by setting the `--issuer-ambient-credentials` flag on
+> the cert-manager controller to true.
 
-cert-manager supports two ways of specifying credentials:
+Here is an example of a `ClusterIssuer` for using Route53 ambient credentials:
 
-- explicit by providing an `accessKeyID` or an `accessKeyIDSecretRef`, and a `secretAccessKeySecretRef`
-- or implicit (using [metadata
-  service](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html)
-  or [environment variables or credentials
-  file](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials).
+```yaml
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-prod
+spec:
+  acme:
+    ...
+    solvers:
+    - dns01:
+        route53 {}:
+```
 
-cert-manager also supports specifying a `role` to enable cross-account access
-and/or limit the access of cert-manager. Integration with
-[`kiam`](https://github.com/uswitch/kiam) and
-[`kube2iam`](https://github.com/jtblin/kube2iam) should work out of the box.
+> ℹ️ Regardless of which ambient mechanism you use, the `route53` section is left empty,
+> because cert-manager can find the credentials, role, and region by looking for environment variables
+> which will be added to the cert-manager Pod.
+
+#### EKS Pod Identity
+
+[EKS Pod Identity](https://docs.aws.amazon.com/eks/latest/userguide/pod-identities.html) is the simplest way to use ambient credentials,
+if you deploy cert-manager on EKS.
+It is a four step process:
+1. [Setup the EKS Pod Identity agent](https://docs.aws.amazon.com/eks/latest/userguide/pod-id-agent-setup.html) in your cluster.
+2. [Assign an IAM role to the cert-manager Kubernetes service account](https://docs.aws.amazon.com/eks/latest/userguide/pod-id-association.html).
+3. Restart the cert-manager Deployment
+   so that the EKS Pod Identity Agent can inject the necessary environment variables into the Pods.
+4. Create a `ClusterIssuer` resource:
+
+   ```yaml
+   apiVersion: cert-manager.io/v1
+   kind: ClusterIssuer
+   metadata:
+     name: letsencrypt-prod
+   spec:
+     acme:
+       ...
+       solvers:
+       - dns01:
+           route53 {}:
```

Comment:
In https://github.com/cert-manager/cert-manager/issues/6909 @mvink-guida reports that Pod Identity works with cert-manager and I have tested it myself on an EKS cluster.

@mvink-guida please read this new section and comment whether it is sufficient: 
 * https://deploy-preview-1555--cert-manager.netlify.app/docs/configuration/acme/dns01/route53/#eks-pod-identity